demo/src/main/java/App.java

```diff
@@ -45,4 +67,11 @@ private static long fib(Long n) throws InterruptedException {
         Thread.sleep(100);
         return fib(n - 1) + fib(n - 2);
     }
+
+    private static class MyStdoutExporter implements Exporter {
+        @Override
+        public void export(Snapshot snapshot) {
+            System.out.printf("Export %d %d%n", snapshot.data.length, snapshot.labels.toByteArray().length);
+        }
+    }
 }
```

Comment:
Nit: Can you set up your editor to automatically add new lines at the end of each file?

Reply:
done